index.html

```diff
@@ -77,13 +78,19 @@
       </main>
       <footer>
         <span>&copy; 2016 Peter Breen</span>
+        <ul class="social-list">
+          <li><a class="icon-github" href="https://github.com/PeterBreen"></a></li>
+          <li><a class="icon-linkedin" href="https://www.linkedin.com/in/peter-breen-96404142"></a></li>
+          <li><a class="icon-flickr3" href="https://www.flickr.com/photos/notpsion"></a></li>
+          <li><a class="icon-wordpress" href="http://peterbreen.info/wp"></a></li>
+        </ul>
       </footer>
     <script src="vendor/jquery.min.js"></script> <!-- TODO: make CDN links when deployed - jquery v2.2.2, handlebars 4.0.5 -->
     <script src="vendor/handlebars.min.js"></script>
     <script src="scripts/portfolio.js"></script>
     <script src="scripts/portfolioView.js"></script>
     <script>
-      Project.fetchAll();
+      Project.fetchAll(portfolioView.initIndexPage);
```

Comment:
fetchAll doesn't currently take a function as a parameter. How can you make this line of code work as it should?